tests/python_package_test/test_sklearn.py

```diff
@@ -922,3 +922,14 @@ def test_continue_training_with_model(self):
         self.assertEqual(len(init_gbm.evals_result_['valid_0']['multi_logloss']), 5)
         self.assertLess(gbm.evals_result_['valid_0']['multi_logloss'][-1],
                         init_gbm.evals_result_['valid_0']['multi_logloss'][-1])
+
+    def test_eval_metrics_lgbmclassifier(self):
```

Comment:
Will it make sense to include the content of this test into already existing huge unit test for metrics as an enhancement for the case of `... invalid multiclass metric is replaced with binary alternative ...`
https://github.com/microsoft/LightGBM/blob/2792923d35e7f3d392474ceffe2185c661047b2e/tests/python_package_test/test_sklearn.py#L503
https://github.com/microsoft/LightGBM/blob/2792923d35e7f3d392474ceffe2185c661047b2e/tests/python_package_test/test_sklearn.py#L749-L753

Or just please move this new test closer to that one.

Reply:
@StrikerRUS yes definitely makes sense. I've moved the test inside of `test_metrics` but placed it next to the section 

> non-default metric with multiple metrics in eval_metric

as it is highly related to that test. Thanks!